src/lambda_codebase/account_processing/configure_account_regions.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+"""
+Takes regions that the account is not-optend into and opts out of them.
+
+"""
+from ast import literal_eval
+
+
+import boto3
+from aws_xray_sdk.core import patch_all
+from logger import configure_logger
+
+patch_all()
+LOGGER = configure_logger(__name__)
+
+
+def get_regions_from_ssm(ssm_client):
+    regions = ssm_client.get_parameter(Name="target_regions")["Parameter"].get("Value")
+    regions = literal_eval(regions)
+    return regions
+
+
+def enable_regions_for_account(
+    account_client, account_id, desired_regions, target_is_different_account
+):
+    list_region_args = {}
+    enable_region_args = {}
+    if target_is_different_account:
+        list_region_args["AccountId"] = account_id
+        enable_region_args["AccountId"] = account_id
+    region_status_response = account_client.list_regions(**list_region_args)
+    region_status = {
+        region.get("RegionName"): region.get("RegionOptStatus")
+        for region in region_status_response.get("Regions")
+    }
+    # Currently no built in paginator for list_regions...
+    # So we have to do this manually.
+    next_token = region_status_response.get("NextToken")
+    if next_token:
+        while next_token:
+            list_region_args["NextToken"] = next_token
+            region_status_response = account_client.list_regions(**list_region_args)
+            next_token = region_status_response.get("NextToken")
+            region_status = region_status | {
+                region.get("RegionName"): region.get("RegionOptStatus")
+                for region in region_status_response.get("Regions")
+            }
+
+    regions_enabled = {}
+    for region in desired_regions:
+        regions_enabled[region] = False
```

Comment:
Region disabling is not currently enabled. (a) because we can't guarantee that there are no resources in that region (b) because of the concerns above. I think opt-in is a more pressing use case for customers.